README.md

```diff
@@ -25,6 +25,10 @@ npm run dev
 # navigate http://localhost:5173/
 ```
 
+### Database/Functions
+
+checkout [`database/README.md`](database/README.md) and [functions/README.md](functions/README.md)
+
```

Comment:
Can we update the readme with proper monorepo behavior? i.e. if I want to run npm commands for one of these functions -- I should `cd` into the appropriate folder I presume?

Reply:
hmm correct, maybe we can come up with a root `npm` run script to manage this. the firebase emulator can handle deploying all local examples so we can make it consistent with one entry point if thats helpful. 


good request to add more documentation and pattern to do this

Reply:
theres a predeploy functionality that automatically runs during the pre-serve step which is cool. so you dont have to CD to deploy those subdirectories to npm install for deploy